sympy/physics/mechanics/tests/test_body.py

```diff
@@ -62,6 +62,8 @@ def test_particle_body():
     assert hasattr(particle_body, 'frame')
     assert hasattr(particle_body, 'masscenter')
     assert hasattr(particle_body, 'mass')
+    assert particle_body.inertia == (Dyadic(0), particle_body.masscenter)
+    assert particle_body.central_inertia == Dyadic(0)
```

Comment:
It seems `.inertia` gives the central inertia, so why are both needed here? What if you want an inertia about another point? Is that part of the API?

Reply:
I see now that RigidBody can be defined with its `.inertia` wrt to any point. Does this work for particle too? A particle can have inertia wrt a point other than its mass center.

Reply:
I've just added a test showing the expected behaviour.